be/compendium/db/models.py

```diff
@@ -24,7 +24,7 @@ class Users(models.Model):
 class Daily_reports(models.Model):
     user_id = models.ForeignKey(Users, on_delete=models.CASCADE)
     progress = models.TextField()
-    problems = models.TextField(default=0)
+    problems = models.TextField()
```

Comment:
since you change the default here, please create another migration for this so that it will take effect in the db as well